models.py

```diff
@@ -0,0 +1,294 @@
+# This is an auto-generated Django model module.
```

Comment:
Would you be able to run pip freeze > requirements.txt so that we can all update our virtual environments to what you have in yours?

Reply:
yes will do